src/web-worker/amp-worker.js

```diff
@@ -103,6 +106,17 @@ class AmpWorker {
       })
       .then((res) => res.text())
       .then((text) => {
+        // Replace sourceMappingUrl with the absolute URL
+        const base = calculateEntryPointScriptBase(
+          loc,
+          useLocal,
+          useRtvVersion
+        );
```

Comment:
Actually, we'll shouldn't even need this. We know the `url` variable above is the URL to the `ww.js` script (or `ww.mjs` if using module build). So the sourcemap is that same URL, plus `.map`:

```suggestion
        const sourceMappingUrl = `${url}.map`;
```